### PR TITLE
A couple of suggestions

### PR DESCRIPTION
**Pull request by Aarononomous**
Here are a couple of suggestions in code form. I'll add comments inline.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -25,19 +25,22 @@
             <div class="increase-tempo-content">+</div>
           </button>
         </div>
-        <div class="triangle">
-          <button class="start-stop">
+        <button class="start-stop">
```
Comment (Aarononomous) [inline index.html:28]: The idea here is that the start and stop images are _within_ the `<button>`. This way we don't need to modify the contents of the button, just toggle the `stop` (or `stopped`?) class.

Then we display, or not, the images with CSS something like

```css
.start-stop start {
  display: block;
}

.start-stop stop {
  display: none;
}

.start-stop.stopped start {
  display: none;
}

/* etc */
```

File: app.js [hunk 2]
```diff
@@ -103,34 +103,24 @@ function updateMetronome() {
...
+  const actual = now - lastClick;
+  const diff = new Intl.NumberFormat("en-US", { signDisplay: "exceptZero" }).format(actual - expected);
+  console.log("click", { expected, actual, diff });
+  lastClick = now;
   timeout = setTimeout(playMetronome, 60000 / tempo);
 }
 
 startStopButton.addEventListener("click", () => {
+  startStopButton.classList.toggle("stopped");
+  isPlaying = !isPlaying;
```
Comment (Aarononomous) [inline app.js:119]: The changes here are just to improve readability. If we can pull these out of the if-else block, let's. (If there were more complex logic where, say, four variables got updated or not, I'd suggest the opposite change!)

File: app.js [hunk 3]
```diff
@@ -103,34 +103,24 @@ function updateMetronome() {
...
 
+let lastClick = performance.now();
 function playMetronome() {
-  console.log("click");
+  const now = performance.now();
+  const expected = 60000 / tempo;
+  const actual = now - lastClick;
+  const diff = new Intl.NumberFormat("en-US", { signDisplay: "exceptZero" }).format(actual - expected);
+  console.log("click", { expected, actual, diff });
+  lastClick = now;
```
Comment (Aarononomous) [inline app.js:113]: Try running this and seeing what happens on your machine. I see sometimes a lag of 10 - 30 ms, sometimes only a couple of millis. That it's consistent is good. That it slows down the actual tempo is bad.

I'd like you to try and resolve this. Let me know if you need some hints!

Don't put off getting clicks playing to work on this. They're orthogonal.

**Review by Aarononomous — COMMENTED** [review]
Great work so far. Just leaving a couple comments. Let me know if you have any questions!

File: app.js (hunk 3 above)
Comment (queeeq) [reply app.js:113]: I will work on this today and tomorrow. Thanks!

**Pull request closed without merging**